tictactoe/tictactoe.js

```diff
@@ -0,0 +1,94 @@
+var origBoard;
+var player;
+const player1   = "X";
+const player2   = "O";
+const winCombos = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [2,4,6],
+]
+const cells = document.querySelectorAll(".cell");
+startGame();
+
+function startGame() {
+    document.querySelector("#endgame").style.display = "none";
```

Comment:
I'd place startup state style in the CSS, then put runtime style changes into the JS.

Reply:
This is so when you click the reset button, it hides the winner data and the reset button from view.